tests/test_app_metadata.py

```diff
@@ -124,5 +197,11 @@ def test_app_metadata(
     # stack begins with index
     idx = int(stack.split(";")[0])
 
+    if runtime == "java":
+        # dont check the jvm flags, they have different values on different machines
```

Comment:
You *do* check them after all, so what you mean is "don't check the JVM flags in direct comparison, as they might change a bit across machines due to ergonomics"